testsupport/wait/member.go

```diff
@@ -1806,24 +1806,39 @@ func (a *MemberAwaitility) verifyUsersRolebindingsWebhookConfig(ca []byte) {
-	assert.Equal(a.T, admv1.SideEffectClassNone, *webhook.SideEffects)
-	assert.Equal(a.T, int32(5), *webhook.TimeoutSeconds)
-	assert.Equal(a.T, admv1.Ignore, *webhook.FailurePolicy)
-	assert.Equal(a.T, admv1.Equivalent, *webhook.MatchPolicy)
-	assert.Equal(a.T, codereadyToolchainProviderLabel, webhook.NamespaceSelector.MatchLabels)
-	assert.Equal(a.T, ca, webhook.ClientConfig.CABundle)
-	assert.Equal(a.T, "member-operator-webhook", webhook.ClientConfig.Service.Name)
-	assert.Equal(a.T, a.Namespace, webhook.ClientConfig.Service.Namespace)
-	assert.Equal(a.T, "/validate-users-rolebindings", *webhook.ClientConfig.Service.Path)
-	assert.Equal(a.T, int32(443), *webhook.ClientConfig.Service.Port)
-	require.Len(a.T, webhook.Rules, 1)
-
-	rule := webhook.Rules[0]
+	require.Len(a.T, actualValWbhConf.Webhooks, 2)
+
+	roleBindingWebhook := actualValWbhConf.Webhooks[0]
+	assert.Equal(a.T, "users.rolebindings.webhook.sandbox", roleBindingWebhook.Name)
+	assert.Equal(a.T, []string{"v1"}, roleBindingWebhook.AdmissionReviewVersions)
+	assert.Equal(a.T, admv1.SideEffectClassNone, *roleBindingWebhook.SideEffects)
+	assert.Equal(a.T, int32(5), *roleBindingWebhook.TimeoutSeconds)
+	assert.Equal(a.T, admv1.Ignore, *roleBindingWebhook.FailurePolicy)
+	assert.Equal(a.T, admv1.Equivalent, *roleBindingWebhook.MatchPolicy)
+	assert.Equal(a.T, codereadyToolchainProviderLabel, roleBindingWebhook.NamespaceSelector.MatchLabels)
+	assert.Equal(a.T, ca, roleBindingWebhook.ClientConfig.CABundle)
+	assert.Equal(a.T, "member-operator-webhook", roleBindingWebhook.ClientConfig.Service.Name)
+	assert.Equal(a.T, a.Namespace, roleBindingWebhook.ClientConfig.Service.Namespace)
+	assert.Equal(a.T, "/validate-users-rolebindings", *roleBindingWebhook.ClientConfig.Service.Path)
+	assert.Equal(a.T, int32(443), *roleBindingWebhook.ClientConfig.Service.Port)
+	require.Len(a.T, roleBindingWebhook.Rules, 1)
+
+	checlusterWebhook := actualValWbhConf.Webhooks[1]
+	assert.Equal(a.T, "users.checluster.webhook.sandbox", checlusterWebhook.Name)
+	assert.Equal(a.T, []string{"v1"}, checlusterWebhook.AdmissionReviewVersions)
+	assert.Equal(a.T, admv1.SideEffectClassNone, *checlusterWebhook.SideEffects)
+	assert.Equal(a.T, int32(5), *checlusterWebhook.TimeoutSeconds)
+	assert.Equal(a.T, admv1.Ignore, *checlusterWebhook.FailurePolicy)
+	assert.Equal(a.T, admv1.Equivalent, *checlusterWebhook.MatchPolicy)
+	assert.Equal(a.T, codereadyToolchainProviderLabel, checlusterWebhook.NamespaceSelector.MatchLabels)
+	assert.Equal(a.T, ca, checlusterWebhook.ClientConfig.CABundle)
+	assert.Equal(a.T, "member-operator-webhook", checlusterWebhook.ClientConfig.Service.Name)
+	assert.Equal(a.T, a.Namespace, checlusterWebhook.ClientConfig.Service.Namespace)
+	assert.Equal(a.T, "/validate-users-checlusters", *checlusterWebhook.ClientConfig.Service.Path)
+	assert.Equal(a.T, int32(443), *checlusterWebhook.ClientConfig.Service.Port)
+	require.Len(a.T, checlusterWebhook.Rules, 1)
+
+	rule := roleBindingWebhook.Rules[0]
```

Comment:
Shouldn't we also check the rules for the `checlusterWebhook`?

Reply:
good point, let me add that